getting-started/release-notes/changes-in-timescaledb-2.md

```diff
@@ -80,40 +80,40 @@ In the rest of this document, we go through each of the features and API changes
 what users migrating from an earlier version of TimescaleDB should consider prior to updating to TimescaleDB 2.0. 
 
 
-# Working with hypertables [](hypertables)
+## Working with hypertables [](hypertables)
 
 In TimescaleDB 2.0, we have made changes to existing APIs for working with hypertables, as well 
 as improvements to the related information views and size functions. These views and functions 
 provide information about basic configuration, partitioning, data chunks, and disk size, and they 
 also have been updated to work for distributed hypertables.
 
 
-## Creating hypertables and changing configuration
+### Creating hypertables and changing configuration
 
 The following APIs to create and configure hypertables have changed:
 
-*   [create_hypertable](https://docs.timescale.com/api#create_hypertable):  The `main_table` parameter has been renamed to `relation`, and additional parameters for distributed hypertables have been added.
-*   [set_chunk_time_interval](https://docs.timescale.com/v2.0/api#set_chunk_time_interval), [set_number_of_partitions](https://docs.timescale.com/v2.0/api#set_number_partitions), [add_dimension](https://docs.timescale.com/v2.0/api#add_dimension):  The `main_table` parameter has changed name to `hypertable`.
+*   [`create_hypertable`](https://docs.timescale.com/api#create_hypertable):  The `main_table` parameter has been renamed to `relation`, and additional parameters for distributed hypertables have been added.
```

Comment:
Minor point, and these will work, but I *believe* these can just use local links `/api#create_hypertable` as opposed to the full URL.  Okay to do these, but just might be easier other way....